Index: ejabberd/src/mod_stream_mgmt.erl
===================================================================
--- ejabberd.orig/src/mod_stream_mgmt.erl
+++ ejabberd/src/mod_stream_mgmt.erl
@@ -523,7 +523,7 @@ route_unacked_stanzas(#{mgmt_state := Mg
 			  Resend when is_boolean(Resend) ->
 			      Resend;
 			  if_offline ->
-			      case ejabberd_sm:get_user_resources(User, Resource) of
+			      case ejabberd_sm:get_user_resources(User, LServer) of
 				  [Resource] ->
 				      %% Same resource opened new session
 				      true;
