From a9c6748ec7831ba5e0323186f148edb78ba9aa05 Mon Sep 17 00:00:00 2001
From: Christophe Romain <christophe.romain@process-one.net>
Date: Mon, 1 Aug 2016 10:55:02 +0200
Subject: [PATCH] Add missing comas in sql statement (#1219)

---
 src/nodetree_tree_sql.erl | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/nodetree_tree_sql.erl b/src/nodetree_tree_sql.erl
index edfdbc1..9f6b6d5 100644
--- a/src/nodetree_tree_sql.erl
+++ b/src/nodetree_tree_sql.erl
@@ -77,9 +77,9 @@ set_node(Record) when is_record(Record, pubsub_node) ->
 	    catch
 	    ejabberd_sql:sql_query_t(
               ?SQL("update pubsub_node set"
-                   " host=%(H)s"
-                   " node=%(Node)s"
-                   " parent=%(Parent)s"
+                   " host=%(H)s,"
+                   " node=%(Node)s,"
+                   " parent=%(Parent)s,"
                    " type=%(Type)s "
                    "where nodeid=%(OldNidx)d")),
 	    OldNidx;
